gradle-enterprise-conventions-plugin/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    `java-gradle-plugin`
+    id("org.jetbrains.kotlin.jvm") version "1.3.72"
+    id("com.gradle.plugin-publish") version "0.12.0"
+    id("maven-publish")
+}
+
+rootProject.group = "org.gradle.enterprise"
+rootProject.version = "0.1-SNAPSHOT"
+
+repositories {
+    jcenter()
+    gradlePluginPortal()
+}
+
+dependencies {
+    val gradleEnterprisePluginVersion = "3.3.4"
+    compileOnly("com.gradle:gradle-enterprise-gradle-plugin:${gradleEnterprisePluginVersion}")
+    implementation(gradleApi())
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
```

Comment:
Is the 'kotlin-bom' needed here? I thought the versions are added by the Kotlin plugin anyway?